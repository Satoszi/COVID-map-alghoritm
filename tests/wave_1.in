8 8 2 20
wave_1
++..++..
||++||++
||||||||
||||||||
||||||||
||||||||
|||++|||
|++..++|