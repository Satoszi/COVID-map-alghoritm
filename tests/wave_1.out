7 7
1 7
0 7
6 6
5 6
4 5
3 5
2 5
7 4
1 4
0 4
6 3
5 3
4 2
3 2
2 2
7 1
1 1
0 1
5 0
